verinfast/cloud/aws/instances.py

```diff
@@ -3,7 +3,7 @@
 
 import boto3
 
-import regions as r
+import cloud.aws.regions as r
```

Comment:
does `import .regions as r` work for you? I'd rather keep it a relative import, so we can continue testing individual files.

Reply:
Nope:
`Traceback (most recent call last):
  File "/home/stconrad/verinfast/verinfast/./agent.py", line 56, in <module>
    from cloud.aws.instances import get_instances as get_aws_instances
  File "/home/stconrad/verinfast/verinfast/cloud/aws/instances.py", line 7
    import .regions as r
           ^
SyntaxError: invalid syntax`

If I understand it correctly, for imports, Python doesn't look in directories, it looks in the sys.path. The main script that was run gets its path added, but the imports (instances.py) in this case, don't know where they are.

Reply:
yeah, that's the frustration I have with testing... I want to test small pieces not wait for the whole thing to run. Duly noted and merging.